<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBIWnd</class>
 <widget class="QWidget" name="CBIWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>过滤</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>筛选：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_BISelectComboBox">
        <item>
         <property name="text">
          <string>显示全部遥信</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示无类型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示A类遥信</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示B类遥信</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示C类遥信</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>属性：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_BIPropertyComboBox">
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>单点硬开入</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>单点软遥信</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>双点遥信</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>名称包含：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_BIInfoLineEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>392</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="m_BITableWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>130</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>属性</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>采样通道</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>防抖时间(ms)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>关联常闭节点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>遥信类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>描述</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
